packages/big-design/src/components/Checkbox/Checkbox.tsx

```diff
@@ -8,6 +8,7 @@ import { uniqueId } from '../../utils';
 import { CheckboxContainer, HiddenCheckbox, StyledCheckbox, StyledLabel } from './styled';
 
 interface Props {
+  hiddenLabel?: boolean;
   isIndeterminate?: boolean;
   label?: React.ReactChild;
```

Comment:
We probably want to add back `label` as required again. See #201 on why we made it optional. Afterwards, you may want to go into the table components and update the Checkboxes accordingly.

Reply:
Yeah, I went back and forth on whether to do that or make it part of another ticket. If this all looks okay, I'll go ahead and update the table component too.